etna/lib/helpers.rb

```diff
@@ -0,0 +1,57 @@
+require_relative './etna/clients'
```

Comment:
If we're adding this from polyphemus, are we also removing it from polyphemus and just using this shared implementation?  Didn't see that in the PR but maybe I'm blind.

Reply:
This is one of those files where I just added it to make local testing inside of `etna` easier. Depending on how you decide to package it up for installation, feel free to remove this and I can plug the workflow back in where it makes sense (polyphemus or other location).

Reply:
And no, you didn't miss it, I didn't remove it from Polyphemus since I wasn't intending this to be a shared implementation moving forward...